src/client/js/client.js

```diff
@@ -151,6 +152,13 @@ define([
                         logger.warn('Lots of persisted objects', numberOfPersistedObjects);
                     }
 
+                    if (gmeConfig.storage.patchRootCommunicationEnabled) {
+                        persisted.rootHash = {

```

Comment:
Btw Plugins, AddOns, client/merge.js, core/users/merge.js are not updated to use the patch-root.